packages/common/src/content/fetch.ts

```diff
@@ -264,13 +266,21 @@ export const fetchHubContent = async (
   setProp("type", type, item);
 
   const model = { item };
-  const enrichments: IItemAndIServerEnrichments = {};
+  const enrichments: IHubEditableContentEnrichments = {};
   if (isHostedFeatureServiceItem(item)) {
     enrichments.server = await getService({
       ...requestOptions,
       url: parseServiceUrl(item.url),
     });
   }
+
+  if (isDownloadSchedulingAvailable(requestOptions)) {
```

Comment:
Shoot, last big thing. Aren't schedules only available for _public_ items? Double check with San, but if so we should probably bake that check into `isDownloadSchedulingAvailable`. 

On that note, I recommend we move the `|| { mode: 'automatic' }` fallback within this if statement so private items (and enterprise items) will have an undefined `schedule`.